locallibrary/locallibrary/settings.py

```diff
@@ -138,12 +132,15 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 
+STATIC_ROOT = BASE_DIR / 'staticfiles'
```

Comment:
đồng nhất tên folder với static url nha em
```suggestion
STATIC_ROOT = BASE_DIR / 'static'
```